server.py

```diff
@@ -488,6 +488,48 @@ async def post_prompt(request):
             else:
                 return web.json_response({"error": "no prompt", "node_errors": []}, status=400)
 
+        @routes.post("/prompt_async")
+        async def post_prompt_async(request):
+            print("yeeehaw")
+            resp_code = 200
+            out_string = ""
+            json_data = await request.json()
+            json_data = self.trigger_on_prompt(json_data)
+
+            if "number" in json_data:
```

Comment:
should factor out the common `/prompt` variant code into a single function, and only contain the unique bits at the end in the individual impls